src/OrbitVulkanLayer/SubmissionTracker.h

```diff
@@ -355,6 +389,15 @@ class SubmissionTracker : public VulkanLayerProducer::CaptureStatusListener {
       for (uint32_t command_buffer_index = 0; command_buffer_index < submit_info.commandBufferCount;
            ++command_buffer_index) {
         VkCommandBuffer command_buffer = submit_info.pCommandBuffers[command_buffer_index];
```

Comment:
Nit: Not sure this is practical here, but when you start having multiple continues in a loop body, I generally find it easier to read when you move the loop body into a function and replace the continues with returns.